_aggregations/metric/extended-stats.md

```diff
@@ -9,8 +9,41 @@ redirect_from:
 
 # Extended stats aggregations
 
-The `extended_stats` aggregation is an extended version of the [`stats`]({{site.url}}{{site.baseurl}}/query-dsl/aggregations/metric/stats/) aggregation. Apart from including basic stats, `extended_stats` also returns stats such as `sum_of_squares`, `variance`, and `std_deviation`.
-The following example returns extended stats for `taxful_total_price`:
+The `extended_stats` aggregation is an extended version of the [`stats`]({{site.url}}{{site.baseurl}}/query-dsl/aggregations/metric/stats/) aggregation. It returns all the metrics returned by `stats`, plus the following:
+
+- Sum of squares
+- Variance
+- Population variance
+- Sampling variance
+- Standard deviation
+- Population standard deviation
+- Sampling standard deviation
+- Standard deviation bounds:
+  - Upper
+  - Lower
+  - Population upper
+  - Population lower
+  - Sampling upper
+  - Sampling lower
+
+The standard deviation and variance are population statistics; they are always equal to the population standard deviation and variance, respectively.
+
+The `std_deviation_bounds` object provides values representing an interval of plus or minus a number of standard deviations from the mean (two by default). This object is always displayed, but is meaningful only for normally distributed data. It's up to you to verify that your data set has a normal distribution.
```

Comment:
~object~

Reply:
@sandeshkr419 Are you suggesting deleting "object" from "The `std_devaition_bounds` object", from "This object" in the second sentence, or both?

More generally, I take it "object" is not generally used to describe a data entity in OpenSearch? What would you use instead?

Reply:
We do describe JSON objects as objects in OpenSearch so this usage is appropriate and is aligned with the terminology in our other documentation. When describing fields within an object, we sometimes create sections titled "x object". I suggest keeping the word "object" here.